spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/utils/jackson/AdminServerModule.java

```diff
@@ -37,6 +44,7 @@ public AdminServerModule(String[] metadataKeyPatterns) {
 
 		addDeserializer(Registration.class, new RegistrationDeserializer());
 		setSerializerModifier(new RegistrationBeanSerializerModifier(new SanitizingMapSerializer(metadataKeyPatterns)));
+		setMixInAnnotation(InstanceEvent.class, InstanceEventMixin.class);
```

Comment:
The InstanceEvent is adapted using a mixin but in the other classes the annotations are put on the classes themself.
I wonder if we shouldn't use a single way of adding thejackson  metadata?

Reply:
I did it because StatusInfo, InstanceId and Info have already some "basic" Jackson annotations direct in the classes.
I'll move the newly added Jackson annotations to Mixin classes.
Additionally I'll try to move the previous existing Jackson annotations too.
Then the domain classes will have no more Jackson specif annotations in.